invenio_communities/fixtures/tasks.py

```diff
@@ -26,11 +27,17 @@
 
 
 @shared_task
-def create_demo_community(data):
+def create_demo_community(data, logo_path=None):
     """Create a demo community."""
     service = current_communities.service
     try:
-        service.create(data=data, identity=system_identity)
+        community = service.create(data=data, identity=system_identity)
+
+        # upload logo for community if provided
+        if logo_path:
+            with open(Path(logo_path), "rb") as filestream:
```

Comment:
```suggestion
            with open(logo_path, "rb") as filestream:
```

minor: wrapping with `Path` doesn't add anything here, since we're not e.g. manipulating the path or checking if the file exists, etc.